py/objint_mpz.c

```diff
@@ -356,9 +356,10 @@ static mpz_t *mp_mpz_for_int(mp_obj_t arg, mpz_t *temp) {
 mp_obj_t mp_obj_int_pow3(mp_obj_t base, mp_obj_t exponent,  mp_obj_t modulus) {
     if (!mp_obj_is_int(base) || !mp_obj_is_int(exponent) || !mp_obj_is_int(modulus)) {
         mp_raise_TypeError(MP_ERROR_TEXT("pow() with 3 arguments requires integers"));
+    } else if (modulus == MP_OBJ_NEW_SMALL_INT(0)) {
+        mp_raise_ValueError(MP_ERROR_TEXT("pow() 3rd argument cannot be 0"));
```

Comment:
This new error string adds quite a bit of code size, for arguably not much gain.

Suggest making it smaller, maybe reuse the divide-by-zero message `"divide by zero"` but keep it as a ValueError?

Reply:
done.
```>>> 1/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: divide by zero
>>> pow(1,1,0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: divide by zero
```